### PR TITLE
Setup GHA.

### PR DESCRIPTION
**Pull request by icemac**

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -4,7 +4,7 @@ Changelog
 4.2 (unreleased)
 ----------------
 
-- Add support for Python 3.7 and 3.8.
+- Add support for Python 3.7 up to 3.9.
```
Comment (mgedmin) [inline CHANGES.rst]: Eh, I'd enumerate them all, in case people want to grep for 3.8 in the changelog.

File: tox.ini [hunk 2]
```diff
@@ -37,7 +40,7 @@ deps =
 commands =
     coverage run -m zope.testrunner --test-path=src []
     coverage html
-    coverage report -m
+    coverage report -m --fail-under=0
```
Comment (mgedmin) [inline tox.ini]: Talk about a low bar :)

**Review by mgedmin — APPROVED** [review]
LGTM

File: CHANGES.rst (hunk 1 above)
Comment (icemac) [reply CHANGES.rst]: Okay.

File: tox.ini (hunk 2 above)
Comment (icemac) [reply tox.ini]: Fixed.

**Comment by icemac** [comment]
Thank you.

**Pull request merged**